addons/auth_signup/models/res_partner.py

```diff
@@ -82,12 +56,7 @@ def _get_signup_url_for_action(self, url=None, action=None, view_type=None, menu
             if signup_type:
                 route = 'reset_password' if signup_type == 'reset' else signup_type
 
-            if partner.sudo().signup_token and signup_type:
-                query['token'] = partner.sudo().signup_token
-            elif partner.user_ids:
-                query['login'] = partner.user_ids[0].login
```

Comment:
Looks like we are loosing that ` query['login'] ` possibility/"feature": Not sure if it was used at some point.

I will run a runbot on master (without this pr) printing the stacktrace when it goes through here to check if we had cases.

Reply:
So, there are cases, it might be worth to investigate them:
https://runbot.odoo.com/runbot/batch/1532641/build/65212483

Meaning, there are cases where `_get_signup_url_for_action` was called without signup_token set, but for partners already having a user ?